gatsby-theme-mds/src/components/LeftNav/LeftNav.js

```diff
@@ -28,6 +29,7 @@ const LeftNav = (props) => {
       onToggle={setExpanded}
       hoverable={false}
     >
+      <Subheading className="title">Introduction</Subheading>
```

Comment:
Please use helper padding classes and heading component with appearance, remove `title` class.